qemu/tests/fio_perf.py

```diff
@@ -86,28 +88,22 @@ def get_version(session, result_file, kvm_ver_chk_cmd,
                           "[Version ide driver format]\n")
 
 
-def clean_tmp_files(session, check_install_fio, tarball,
-                    os_type, guest_result_file, fio_path, timeout):
+def clean_tmp_files(session, check_install_fio, os_type, guest_result_file, timeout):
     """
     del temporary files in guest
 
     :param session: VM session
     :param check_install_fio: check if fio is installed in guest
-    :param tarball: fio tar package
     :param os_type: guest typet
     :param guest_result_file: fio result file in guest
-    :param fio_path: path of fio source code
     :param timeout: Timeout in seconds
     """
 
     if not session.cmd_status(check_install_fio):
```

Comment:
Suggest check if  `fio.fio_path` exist in guest  or not.  Drop `check_install_fio`